src/extras/geometries/SphereBufferGeometry.js

```diff
@@ -30,10 +30,12 @@ THREE.SphereBufferGeometry = function ( radius, widthSegments, heightSegments, p
 	thetaStart = thetaStart !== undefined ? thetaStart : 0;
 	thetaLength = thetaLength !== undefined ? thetaLength : Math.PI;
 
+	var isSector = thetaStart > 0 || thetaStart + thetaLength < Math.PI;
```

Comment:
Not necessarily...?

What if, for whatever reason, `thetaStart` is `Math.PI`? Think about someone using the editor and changing only the `thetaStart` value.